lncfg/db.go

```diff
@@ -338,6 +348,16 @@ func (db *DB) GetBackends(ctx context.Context, chanDBPath,
 		}
 		closeFuncs[NSTowerServerDB] = etcdTowerServerBackend.Close
 
+		etcdPeerStorageBackend, err := kvdb.Open(
+			kvdb.EtcdBackendName, ctx,
+			db.Etcd.CloneWithSubNamespace(NSPeerStorageDB),
+		)
+		if err != nil {
+			return nil, fmt.Errorf("error opening etcd "+
+				"peer storage DB: %v", err)
```

Comment:
is this one of those situations where we need `%w` for errors?

Reply:
Yeah I copied this from other DBs that have been created here,  they used %v, not sure about the best practice for this scenario

Reply:
Yeah stick to convention then that's fine. I was just vaguely remembering a global search and replace PR from @ffranr but I'll admit I'm not totally sure when and why we choose `%w` over `%v`

Reply:
We generally want _all_ errors to be wrapped. There'll be a PR that updates them all (https://github.com/lightningnetwork/lnd/pull/8611). So new code should always wrap.

Reply:
This still needs to be addressed.